database/sql/V048__create_settlement_rewards.sql

```diff
@@ -0,0 +1,44 @@
+-- Populated before settlement occurs on chain
```

Comment:
Ubernit: It would be nice to add a description header to this migration script (since it does so much).

Reply:
https://github.com/cowprotocol/services/pull/1283